docs/resources/fabric_cloud_router.md

```diff
@@ -60,7 +55,7 @@ resource "equinix_fabric_cloud_router" "new_cloud_router"{
 
 - `description` (String) Customer-provided Fabric Cloud Router description
 - `href` (String) Fabric Cloud Router URI information
-- `order` (Block Set, Min: 1, Max: 1) Order information related to this Fabric Cloud Router (see [below for nested schema](#nestedblock--order))
```

Comment:
Min:1 will be lost. Not included in spec? Intended? (perhaps the documentation template isn't reflecting it)

Reply:
Order is optional so it shouldn't have a minimum of 1 anyway. This is intended.